src/ui/Navigation/components/Drawer/Navigation/components/MenuItem/MenuItem.tsx

```diff
@@ -0,0 +1,136 @@
+import { ExpandCollapse, Icon, useTheme, useTypeColorToColorMatch } from '@orfium/ictinus';
+import { BASE_SHADE } from '@orfium/ictinus/dist/theme/palette';
+import React from 'react';
+import { useLocation, useRouteMatch } from 'react-router-dom';
+import FlippableArrow from '../../../../../../FlippableArrow';
+import { MenuItem as MenuItemType } from '../../../../../types';
+import {
+  ArrowContainer,
+  Bullet,
+  ExpandCollapseWrapper,
+  MenuIcon,
+  MenuItemButton,
+  MenuItemText,
+  MenuLink,
+} from './MenuItem.styles';
+
+type Props = {
+  /** Defines if the menu item is expanded */
+  expanded: boolean;
+  toggleMenuItem: (newUrl: string) => void;
+  item: MenuItemType;
+};
+
+function MenuItemContent(props: { expanded: boolean; item: MenuItemType; isSubMenu?: boolean }) {
+  const { item, expanded, isSubMenu = false } = props;
+  const match = useRouteMatch(item.url);
+  const theme = useTheme();
+  const { calculateColorBetweenColorAndType } = useTypeColorToColorMatch();
+  const { shade } = calculateColorBetweenColorAndType('', 'primary');
+
+  const isCurrent = !!match;
+  const hasSubMenus = item.options.length > 0;
+  const color = isCurrent
+    ? theme.utils.getColor('blue', 600)
+    : theme.utils.getColor('lightGrey', 650);
+
+  return (
+    <React.Fragment>
+      <MenuIcon hidden={isSubMenu ? (isCurrent ? false : true) : false}>
```

Comment:
all constants here are booleans so we can return them, like the below I think

```suggestion
      <MenuIcon hidden={isSubMenu && !isCurrent}>
```